### ISSUE TITLE
ssd checkpoint trained on which pascal dataset?

### ISSUE THREAD

**Issue by ambikaverma** [tryolabs/luminoth#197]
Hi

I saw the following blogpost -
http://luminoth.readthedocs.io/en/latest/usage/checkpoints.html

which mentions an SSD model pre-trained on pascal dataset to be downloaded as a checkpoint and used.
Could you share which pascal dataset is it trained on? 
Is it VOC 2007 or something else? Where can I find this information.
Thanks!

**Comment by joaqo** [comment]
Hey! We use 2007+2012 to train in order to compare with the paper's results. If I recall correctly we achieve around 75.6 mAP on the 2007 test set as opposed to the 77.2 the authors quote on the paper for the same metric.
